src/octopeer-github/main/MainController.ts

```diff
@@ -239,4 +235,10 @@ class MainController implements OptionsObserver {
         return this.database[user][prUrl];
     }
 
+    private getCurrentTab(callback: (tab: Tab) => void) {
```

Comment:
Documentation s.v.p. :innocent: 
At the moment it's the only function without documentation. :P